### PR TITLE
Add Bryan's details

### LINKED ISSUE #71: Add Bryan's details to AboutUs

### PR DESCRIPTION
**Pull request by bryancheny**
Closes #71

### CODE REVIEW

**Review by dwsc37 — APPROVED** [review]
LGTM, but hold on merging for now

File: docs/AboutUs.md [hunk 1]
```diff
@@ -21,6 +21,16 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
...
 
+### Chen Yu Bryan
+
+<img src="images/bryancheny.png" width="200px">
+
+[[github](https://github.com/bryancheny)]
+[[portfolio](team/johndoe.md)]
+
+* Role: Developer
+* Responsibilities: Documentation + Testing
```
[the remark this answers is not in the document]
Comment (dwsc37) [reply docs/AboutUs.md]: I have updated it

**Review by dwsc37 — CHANGES_REQUESTED** [review]
Just need to add back the photo

**Review by dwsc37 — APPROVED** [review]
LGTM

**Review by audipras — APPROVED** [review]

**Pull request merged**